pom.xml

```diff
@@ -108,7 +108,7 @@
         <netty3.version>3.10.6.Final</netty3.version>
         <netty4.version>4.1.108.Final</netty4.version>
         <postgresql.version>42.7.2</postgresql.version>
-        <protobuf.version>3.24.0</protobuf.version>
+        <protobuf.version>4.28.2</protobuf.version>
```

Comment:
if the goal is to address the CVE, can we use 3.25.5 instead? 4.x is a big jump, which may require more validation.